psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -2411,16 +2418,18 @@ void RDFMP2::form_Z()
         Dtemp->scale(0.5);
 
         // Add in the reference contribution
+
         for (int i = 0; i < nocc; ++i)
             Dtemp->add(i, i, 1.0);
-        Ca_ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
-        epsilon_a_ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
+        Ca__ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
+        epsilon_a__ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
```

Comment:
I would think that we actually want the non relaxed natural orbitals for better guesses downstream. I wouldn't add a Ca__ attribute as it's difficult to tell between Ca_ and Ca__.